nuxeo-services/nuxeo-platform-convert/src/main/java/org/nuxeo/ecm/platform/convert/ooomanager/OOoManagerService.java

```diff
@@ -28,7 +28,7 @@
 /**
  * OOoManagerService can either start or stop OpenOffice pool server and return an OfficeDocumentConverter.
  *
- * @author Laurent Doguin
+ * @deprecated Since 8.4. Use 'soffice' with {@link org.nuxeo.ecm.core.convert.plugins.CommandLineConverter} instead
  */
 public interface OOoManagerService {
```

Comment:
Use both `@Deprecated` Javadoc and class annotations (see https://doc.nuxeo.com/display/CORG/Java+Code+Style)